parser/expandpath_test.go

```diff
@@ -0,0 +1,57 @@
+package parser
+
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+)
+
+func mockCurrentUser() (*user.User, error) {
```

Comment:
```suggestion
func newTestUser() (*user.User, error) {
```

Also, is a function required here?

Reply:
I'll put these into anonymous functions. This is for mocking out `user.Current()` from the `os/user` package, so I think renaming it to `newTestUser()` actually creates more confusion.